components/math/multiplicationConnect/GameBoard.tsx

```diff
@@ -1,14 +1,16 @@
+import { FieldNode } from "graphql";
 import React, { FC, useState } from "react";
 import GameBoardBlock from "./GameBoardBlock";
 
 interface GameBoardProps {
   grid: GameBoardBlock[];
+  blockClick(block: GameBoardBlock): void;
+  isPlayerOne: boolean;
 }
 
-const GameBoard: FC<GameBoardProps> = ({ grid }) => {
+const GameBoard: FC<GameBoardProps> = ({ grid, blockClick, isPlayerOne }) => {
```

Comment:
Nice, the grid and isPlayerOne is something you will later refactor to a redux state.

This is great for now. You're currently doing what we call prop drilling. It's a valid technique, but can get messy with more complex components.

That's why we use redux to simplify this state management.